benchmarks/src/h2o.rs

```diff
@@ -64,15 +62,15 @@ pub struct RunOpt {
     /// Path to data files (parquet or csv), using , to separate the paths
     /// Default value is the small files for join x table, small table, medium table, big table files in the h2o benchmark
     /// This is the small csv file case
-    #[structopt(
-        short = "join-paths",
```

Comment:
👍🏻 
This seems to have been a bug in the old impl. I guess structopt ignored it and [used](https://docs.rs/structopt/latest/structopt/#specifying-argument-types) the first letter of the field name (`j`)
It is good that Clap uses a char for short!